### ISSUE TITLE
Issue importing procedures

### ISSUE THREAD

**Issue by Mithandill** [MCreator-Examples/Extra-Resources#1]
I am trying to import the smooth structure spawning procedure from the unziped file provided on github, but only Local variables are imported and no programming blocks
![Capture d’écran 2021-10-18 174537](https://user-images.githubusercontent.com/78301333/137764966-6d132672-9f8d-49ad-9029-ca9f2f6328a8.jpg)

**Comment by northwesttreesgaming** [comment]
Thanks for letting me know.
I talked to the MCreator team (Goldorion) and he said string variables where changed in 2021.2, most likely this broke the procedure when they are imported.

I have just spent about a hour watching the video and putting together the procedure again I should have it reading in a moment. I left the procedures for Pre 2021.2 just incase there is some people still using versions before 2021.2.

I will mark this issue as closed once I have uploaded the new download.
